Taskfile.yaml

```
@@ -16,30 +13,34 @@
     dir: "{{ .TASKFILE_DIR }}"
     cmds:
       - echo "Completing Pre-Start Environment Check For Development..."
-      - export DJANGO_CONFIGURATION=Development && export DJANGO_SETTINGS_MODULE=config.settings && doppler run  -- python3 run/manage.py runserver 
+      - export DJANGO_SETTINGS_MODULE=config.settings && doppler run  -- python3 run/manage.py runserver
     interactive: false
     silent: false
 
-
   install:
-    deps: [venv]
+    deps: [ venv ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)